packages/venia-ui/lib/components/Gallery/__tests__/__snapshots__/gallery.spec.js.snap

```diff
@@ -6,17 +6,14 @@ exports[`renders if \`items\` is an array of objects 1`] = `
 >
   <div>
     <div>
-      <img
-        alt=""
-        height="375"
-        src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAQAAAAFCAQAAADIpIVQAAAADklEQVR42mNkgAJGIhgAALQABsHyMOcAAAAASUVORK5CYII="
-        width="300"
-      />
       <img
         alt="Test Product 1"
+        className="undefined undefined"
```

Comment:
TODO

Reply:
This occurs because `mergeClasses` and `defaultClasses` are not available when running unit tests.

Stopping to debug on this line:
```js
const classes = mergeClasses(defaultClasses, propsClasses);
```

shows

![Screen Shot 2019-10-10 at 1 36 36 PM](https://user-images.githubusercontent.com/13182778/66597384-7abd6400-eb64-11e9-8619-35355c313f49.png)

Therefore on a later line,

```js
const imageClass =
        classes.root + ' ' + (isLoaded ? classes.loaded : classes.notLoaded);
```

we're left with `undefined undefined`.